### PR TITLE
perldelta for 91e15e585c79, reinstate apostrophe package separator

### PR DESCRIPTION
**Pull request by tonycoz**
It's in incompatible changes, it doesn't really belong in Core Enhancements.

---------------------------------------------------------------------------------
* This set of changes is a perldelta entry, and it is included.

### CODE REVIEW

File: pod/perldelta.pod [hunk 1]
```diff
@@ -45,6 +45,35 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 Apostrophe is again recognized as a global name separator
+
+This was deprecated in Perl 5.38 and removed as scheduled in perl
+5.41.3, after some discussion has been reinstated by default.
```
Comment (jkeenan) [inline pod/perldelta.pod]: s/after/but after/ for better flow of thought.

File: pod/perldelta.pod [hunk 2]
```diff
@@ -45,6 +45,35 @@ XXX For a release on a stable branch, this section aspires to be:
...
+
+If you want to disable use within your own code you can explicitly
+disable the feature:
+
+  no feature "apostrophe_as_package_separator";
+
+Note that disabling this feature only prevents use of apostrophe as a
+package separator within code, unlike with the reverted removal
+symbolic references still treat C<'> as C<::> with the feature
+disabled:
```
Comment (jkeenan) [inline pod/perldelta.pod]: I'm confused by the flow of the previous sentence.  Should it be `unlike with the reverted removal of symbolic references ...`?  Or perhaps, `unlike with the reverted removal, symbolic references ...`

**Review by jkeenan — CHANGES_REQUESTED** [review]

File: pod/perldelta.pod (hunk 1 above)
Comment (tonycoz) [reply pod/perldelta.pod]: added

File: pod/perldelta.pod (hunk 2 above)
Comment (tonycoz) [reply pod/perldelta.pod]: removed the mention of the reverted removal

**Review by jkeenan — CHANGES_REQUESTED** [review]
One more nit; otherwise okay by me.

**Comment by tonycoz** [comment]
> One more nit; otherwise okay by me.

I don't see any comment pointing out that nit.

File: pod/perldelta.pod [hunk 3]
```diff
@@ -45,6 +45,34 @@ XXX For a release on a stable branch, this section aspires to be:
...
+feature which is enabled by default, but is disabled from the 5.41
+feature bundle onwards.
+
+If you want to disable use within your own code you can explicitly
+disable the feature:
+
+  no feature "apostrophe_as_package_separator";
+
+Note that disabling this feature only prevents use of apostrophe as a
+package separator within code, symbolic references still treat C<'> as
```
Comment (jkeenan) [inline pod/perldelta.pod]: s/code,/code;/

Semicolon better to separate two independent clauses.  (Somehow this didn't get posted earlier.)

Comment (tonycoz) [reply pod/perldelta.pod]: TIL, I only remembered `;` to separate colon introduced lists, I didn't know (perhaps forget) the independent clauses use.

**Comment by tonycoz** [comment]
Moved the change to `perl5416delta.pod`.

**Pull request merged**